src/GameObject.h

```diff
@@ -95,6 +96,9 @@ class GameObject {
 
 	// Physics Component that handles collision reactions
 	PhysicsComponent *physics_;
+
+	//Currenlty active shader

```

Comment:
Think this might be redundant with the shaderName_ in RenderComponent.h; I'd say add a getter in RenderComponent and then have the GameObject call that get function in it's getShader method (like I said before, w/o message passing there's gonna be some ugly "cross-talk" between components, but hopefully we can at least keep data separate).